src/games/strategy/engine/framework/ProcessRunnerUtil.java

```diff
@@ -35,7 +37,7 @@ public static void populateBasicJavaArgs(final List<String> commands, final Stri
     // add 64m to that to
     // compensate
     // final long maxMemory = ((long) (Runtime.getRuntime().maxMemory() * 1.15) + 67108864);
```

Comment:
Those comments should get removed.
I think the reason for th 12% drop is that the JVM requires some memory for itself...